lesson3task/src/Main.java

```diff
@@ -0,0 +1,66 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Scanner sym = new Scanner(System.in);
+        short symbol;
+        System.out.println("What you would like to do?");
+        System.out.println("If you want to plus number - enter 1, minus - 2, multiplication - 3, division - 4, modul - 5:");
+        symbol = sym.nextShort();
+
+
+        if ( symbol == 1 ) {
+            Scanner num = new Scanner(System.in);
+            int first, second, result;
+            System.out.println("Enter first number: ");
+            first = num.nextInt();
+            System.out.println("Enter second number: ");
+            second = num.nextInt();
+            result = first + second;
```

Comment:
краще винести вище так як ця частина всюди повторюється 
```suggestion
        symbol = sym.nextShort();
   Scanner num = new Scanner(System.in);
            int first, second, result;
            System.out.println("Enter first number: ");
            first = num.nextInt();
            System.out.println("Enter second number: ");
            second = num.nextInt();

        if ( symbol == 1 ) {
            result = first + second;
```